[2024-02-11 21:28:34,662] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.get_report scheduled__2024-02-10T21:21:41.485949+00:00 [queued]>
[2024-02-11 21:28:34,667] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: sales_mart.get_report scheduled__2024-02-10T21:21:41.485949+00:00 [queued]>
[2024-02-11 21:28:34,667] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-11 21:28:34,667] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-02-11 21:28:34,667] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-11 21:28:34,672] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): get_report> on 2024-02-10 21:21:41.485949+00:00
[2024-02-11 21:28:34,675] {standard_task_runner.py:52} INFO - Started process 32298 to run task
[2024-02-11 21:28:34,677] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'sales_mart', 'get_report', 'scheduled__2024-02-10T21:21:41.485949+00:00', '--job-id', '772', '--raw', '--subdir', 'DAGS_FOLDER/sprint3 copy.py', '--cfg-path', '/tmp/tmpltl53fg3', '--error-file', '/tmp/tmp56og5_c2']
[2024-02-11 21:28:34,677] {standard_task_runner.py:80} INFO - Job 772: Subtask get_report
[2024-02-11 21:28:34,713] {task_command.py:369} INFO - Running <TaskInstance: sales_mart.get_report scheduled__2024-02-10T21:21:41.485949+00:00 [running]> on host 2a8d1eb5116f
[2024-02-11 21:28:34,756] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=student@example.com
AIRFLOW_CTX_DAG_OWNER=student
AIRFLOW_CTX_DAG_ID=sales_mart
AIRFLOW_CTX_TASK_ID=get_report
AIRFLOW_CTX_EXECUTION_DATE=2024-02-10T21:21:41.485949+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-02-10T21:21:41.485949+00:00
[2024-02-11 21:28:34,757] {logging_mixin.py:115} INFO - Making request get_report
[2024-02-11 21:28:35,459] {logging_mixin.py:115} INFO - Response is b'{"task_id":"MjAyNC0wMi0xMVQyMToyODozMwlkYW1pci1rYWxpbmluNw==","status":"RUNNING","data":{"report_id":null,"start_day":null,"end_date":null,"s3_path":{}}}'
[2024-02-11 21:28:45,741] {logging_mixin.py:115} INFO - Response is b'{"task_id":"MjAyNC0wMi0xMVQyMToyODozMwlkYW1pci1rYWxpbmluNw==","status":"RUNNING","data":{"report_id":null,"start_day":null,"end_date":null,"s3_path":{}}}'
[2024-02-11 21:28:56,095] {logging_mixin.py:115} INFO - Response is b'{"task_id":"MjAyNC0wMi0xMVQyMToyODozMwlkYW1pci1rYWxpbmluNw==","status":"RUNNING","data":{"report_id":null,"start_day":null,"end_date":null,"s3_path":{}}}'
[2024-02-11 21:29:06,584] {logging_mixin.py:115} INFO - Response is b'{"task_id":"MjAyNC0wMi0xMVQyMToyODozMwlkYW1pci1rYWxpbmluNw==","status":"RUNNING","data":{"report_id":null,"start_day":null,"end_date":null,"s3_path":{}}}'
[2024-02-11 21:29:16,808] {logging_mixin.py:115} INFO - Response is b'{"task_id":"MjAyNC0wMi0xMVQyMToyODozMwlkYW1pci1rYWxpbmluNw==","status":"RUNNING","data":{"report_id":null,"start_day":null,"end_date":null,"s3_path":{}}}'
[2024-02-11 21:29:27,175] {logging_mixin.py:115} INFO - Response is b'{"task_id":"MjAyNC0wMi0xMVQyMToyODozMwlkYW1pci1rYWxpbmluNw==","status":"RUNNING","data":{"report_id":null,"start_day":null,"end_date":null,"s3_path":{}}}'
[2024-02-11 21:29:35,392] {local_task_job.py:220} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2024-02-11 21:29:35,393] {process_utils.py:125} INFO - Sending Signals.SIGTERM to group 32298. PIDs of all processes in the group: [32298]
[2024-02-11 21:29:35,393] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 32298
[2024-02-11 21:29:35,393] {taskinstance.py:1541} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-02-11 21:29:35,419] {process_utils.py:75} INFO - Process psutil.Process(pid=32298, status='terminated', exitcode=0, started='21:28:34') (32298) terminated with exit code 0
